instructure/canvas-android
# [MBL-15351][Student][Teacher] Email notification preferences

Test plan:
- Check the email notification preferences in both apps. Compare it with the settings on the web to see if the correct data is shown.
- Verify that changing a notification setting will change that setting on the server as well.

refs: MBL-15351
affects: Student, Teacher
release note: Added email notification preferences to the settings.
